sdk/communication/AzureCommunicationChat/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.1.0 (Unreleased)
```

Comment:
Are you planning to release this as a preview or just a minor bump to the GA SDK?

Reply:
Beta release, as per the other languages, updated now - thanks for pointing this out.